### DISCUSSION TITLE
Update entity ID across configuration

### DISCUSSION THREAD

**Discussion by L0bit0** [frenck/spook#813] [Ideas] [answered]
Hi

I see the service to rename an entity add some value and facilitate renaming entities, especially if you will do a kind of massive.

Anyway, you still have to manually update any other place where this entity is used (dashboards, automations...)

Would be great that the service was able to find entity on the configuration files and also change it.

Thanks in advance

**Comment by frenck** [comment, marked as answer]
> Would be great that the service was able to find entity on the configuration files

It already does that, it will raise repair issues

> and also change it.

It can't do that. Mostly, because YAML can be manually managed, processing them programmatically may result in loss of code comments, YAML-anchor usage, and loss of personal preference in code formatting.

I really want to avoid getting myself burned when touching that stuff automatically.

Nevertheless, thanks for the suggestion 👍 

../Frenck

**Reply by L0bit0** [reply]
thanks it make sense...
